lib/ParseSIL/ParseSIL.cpp

```diff
@@ -1019,7 +1019,34 @@ static bool parseSymbolicValue(SymbolicValue &value, SILParser &SP,
     SILLocation funcLoc = RegularLocation(P.getEndOfPreviousLoc());
     if (SP.parseSILFunctionRef(funcLoc, func))
       return true;
-    value = SymbolicValue::getFunction(func);
+
+    Identifier subConventionId;
+    FunctionSubConvention subConvention;
+    if (!P.Tok.is(tok::l_paren)) {
```

Comment:
Please use `isNot` instead of inverting `is`.

Reply:
done